### PR TITLE
feat: add Pest component

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

File: src/Components/Pest/Installer.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    protected $name = 'install:pest';
+
+    /** {@inheritdoc} */
+    protected $description = 'Pest: Use the Pest testing framework for your test suite';
+
+    /** {@inheritdoc} */
+    public function install(): void
+    {
+        $this->remove('phpunit/phpunit', true);
+        $this->require('pestphp/pest', true);
```
Comment (nunomaduro) [inline src/Components/Pest/Installer.php:31]: Would be cool to also replace the entire test suite by a pest one.

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: src/Components/Pest/Installer.php (hunk 1 above)
Comment (owenvoke) [reply src/Components/Pest/Installer.php:31]: That would be cool, but not sure it'd be possible if someone is using the Pest component on an existing app.

Comment (nunomaduro) [reply src/Components/Pest/Installer.php:31]: Well, you can just ask: "Wish to replace the current test suite by a ..."?

Comment (owenvoke) [reply src/Components/Pest/Installer.php:32]: How about what I've just pushed (9cfa4102810196d96a2a211316a313b32818147a)?

Comment (owenvoke) [reply src/Components/Pest/Installer.php:32]: Sorry, @nunomaduro. Do you think this seems ok now? 🤔

**Comment by nunomaduro** [comment]
This one no longer makes sense.

**Pull request merged**